Parser/Client/Browser.php

```diff
@@ -187,6 +188,7 @@ class Browser extends AbstractClientParser
         'MT' => 'Mint Browser',
         'MX' => 'Maxthon',
         'MY' => 'Mypal',
+        'MR' => 'Monument Browser',
```

Comment:
Is that browser based on Chromium or does the useragent only contain `Chrome` for other reasons? If so we should add it to the browser family.

Reply:
based on chrome
preview
https://user-images.githubusercontent.com/1337066/98821409-928aaa00-2440-11eb-9d9e-3895f2d85e60.png